sherpa/models/regrid.py

```diff
@@ -460,22 +371,18 @@ def overlaps(self, other):
 
     @property
     def grid(self):
-        """
-        Return the grid representation of this dataset. The grid is always a tuple, even if the
-        dataset is 1-D and not integrated. This is due to the existing architecture of Sherpa's
-        model classes and the fact that there is no signature difference among 1-D and 2-D models,
-        as 1-D models can receive 1 or 2 arrays and 2-D models can receive 2 or 4 arrays.
+        """The grid representation of the space.
 
-        The x and y arrays in the grid are one-dimentional representations of the meshgrid obtained
-        from the x and y axis arrays, as in `numpy.meshgrid(x, y)[0].ravel()`
+        The x and y arrays in the grid are one-dimensional
+        representations of the meshgrid obtained from the x and y axis
+        arrays, as in `numpy.meshgrid(x, y)[0].ravel()`
 
         Returns
         -------
         tuple
-            A tuple representing the x and y axes. The tuple will contain four arrays if the dataset is
-            integrated, two otherwise.
-        """
+            A two (x, y) or 4 (x, y, xhi, yhi) tuple.
```

Comment:
```suggestion
            A two element (x, y) or 4 element (x, y, xhi, yhi) tuple.
```
At first, I thought I'll get two (x,y) tuples or even 4 tuples. Then I wondered what those 4 tuples mean. I suggest a rewording to avoid any confusion.